modules/gdscript/editor/script_templates/CharacterBody2D/basic_movement.gd

```diff
@@ -5,26 +5,26 @@ extends _BASE_
 
 const SPEED = 300.0
 const JUMP_VELOCITY = -400.0
+const FRICTION = 600.0
 
-# Get the gravity from the project settings to be synced with RigidBody nodes.
+# Get the gravity from the project settings, to allow syncing with RigidBody2D nodes.
 var gravity: int = ProjectSettings.get_setting("physics/2d/default_gravity")
```

Comment:
Also, not a problem with this PR, but the type here should be `float` not `int`, since that's how the value is defined and it's always used in a float context.

Reply:
Good change for consistency.